presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,143 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds`. ({issue}`4344`)
+* Add support for {func}`extract` for `timestamp(p) with time zone` with values of `p` other than 3. ({issue}`4867`)
+* Add support for correlated subqueries in recursive queries. ({issue}`4877`)
+* Add support for variable precision `TIME WITH TIME ZONE` type. ({issue}`4905`)
+* Remove support for political time zones in `TIME WITH TIME ZONE` type. ({issue}`191`)
+* Expose physical input bytes read statistics. ({issue}`4600`}
+* Report dynamic filters stats. ({issue}`4440`)
```

Comment:
I don't thin this is user visible

Reply:
It's part of query stats JSON. Is that considered user visible?

Reply:
The query stats JSON is an implementation detail, not meant for general user consumption.